lib/travis/api/v3/queries/stages.rb

```diff
@@ -0,0 +1,12 @@
+module Travis::API::V3
+  class Queries::Stages < Query
+
+    def find(build)
+      sort filter(build.stages)
+    end
+
+    def filter(list)
```

Comment:
Will there be any filtering on this? Do we need the filter method if we aren't filtering? Or does this override a default filter method from Query?